foundation_cms/templates/patterns/pages/nothing_personal/home_page.html

```diff
@@ -0,0 +1,7 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <div class="grid-container">
+    <h1> {{ page.title }}</h1>
+  </div>
+{% endblock content %}
```

Comment:
Very basic template for the NP homepage since it wont be public facing until we actually implement it as a team